target/linux/ath79/generic/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -65,7 +65,12 @@ case "$board" in
 		[ "$PHYNBR" -eq 1 ] && \
 			mtd_get_mac_ascii u-boot-env ethaddr > /sys${DEVPATH}/macaddress
 		;;
-	zyxel,nwa1121-ni)
-		mtd_get_mac_text mib0 0x4b > /sys${DEVPATH}/macaddress
+	zyxel,nwa1121-ni|\
+	zyxel,nwa1123-ni)
+		[ "$PHYNBR" -eq 0 ] && \
+			mtd_get_mac_text mib0 0x4b > /sys${DEVPATH}/macaddress
+		[ "$PHYNBR" -eq 1 ] && \
+			mtd_get_mac_text mib0 0x66 > /sys${DEVPATH}/macaddress
+		;;
```

Comment:
Why this sudden change? And apart from that, this is broken due to the duplicate closing sign.

Reply:
Is there any preferred way of incrementally adding cases for devices separated into multiple commits?
I will re-add `[ "$PHYNBR" -eq 0 ] && \` to the NWA1121, so that fewer changes are being introduced by the follow-up commit.
Is it generally acceptable in the first place to have a "redundant" `-eq` here (for NWA1121 that only has one wifi PHY), when it reduces the amount of case blocks?

Reply:
I think adding conditions that will never apply here will cause confusion. An additional device case does not really hurt, but will make things clear for everybody.